fhir-model/src/test/java/com/ibm/fhir/model/util/test/RandomKeyGeneratorTest.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * (C) Copyright IBM Corp. 2019
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.model.util.test;
+
+import static org.testng.Assert.assertFalse;
+
+import org.testng.annotations.Test;
+
+import com.ibm.fhir.model.util.FHIRUtil;
+
+
+public class RandomKeyGeneratorTest {
+
+    @Test
+    public void testGetRandomKey() {
+        String output = FHIRUtil.getRandomKey("AES");
+        assertFalse(output.isEmpty());
```

Comment:
Do you dont have the value to verify in this?
Maybe is return the wrong value but in this Test is pass becouse is not empty.

Reply:
The result is a random value.